### ISSUE TITLE
Possible on-line collaboration tools

### ISSUE THREAD

**Issue by gdevenyi** [gvwilson/collaborative-lesson-development#10]
https://www.sharelatex.com/
https://www.overleaf.com/

**Comment by gdevenyi** [comment]
Google Docs
Github + Markdown
https://hackmd.io/

**Comment by DamienIrving** [comment]
https://www.authorea.com/
https://www.gitbook.com/

**Comment by ianmilligan1** [comment]
We have even done some "shared Word files" in a Dropbox and coordinating via a chat client (wrote a book that way!). The tools weren't the best, but it worked for people. Some tools can scare collaborators, i.e. GitHub (I am guilty of scaring others..!).

**Comment by raynamharris** [comment]
@ianmilligan1 makes an interesting point. I'm currently collaborating with some people who prefer Dropbox. I've tried to convince them to more to a more version-control friendly option, but it is like pulling teeth to get them to try a new technology.

**Comment by ianmilligan1** [comment]
I guess having spanned communities (i.e. my computer science colleagues use one thing, my DH colleagues another, my LIS another, and humanists another), I've become pretty platform agnostic. Going to something technically complex can really alienate people, even in a technical lesson development space!

**Comment by gvwilson** [comment]
@ianmilligan1 can you summarize what each of those communities uses, because I think that would be a lot more interesting than a description of any one set of choices.

**Comment by DamienIrving** [comment]
@gvwilson @ianmilligan1 Has this topic been covered enough in rule 5, or do we need a more in-depth discussion of collaboration tools in the mechanics section?

**Comment by ianmilligan1** [comment]
I think we've got enough in rule 5 so we aren't bogged down in the details?
